src/status_im/ui/components/topnav.cljs

```diff
@@ -40,7 +41,11 @@
                           :on-press #(do
                                        (re-frame/dispatch [:mark-all-activity-center-notifications-as-read])
                                        (if (and @config/new-ui-enabled? @config/new-activity-center-enabled?)
-                                         (re-frame/dispatch [:navigate-to :activity-center])
+                                         (re-frame/dispatch [:show-popover {:view            :activity-center
+                                                                            :blur-view?      true
+                                                                            :blur-view-props {:blurAmount 20
```

Comment:
You can safely use kebab case for props `:blurAmount` and `:blurType`.

Reply:
resolved